src/convertToObject.js

```diff
@@ -6,7 +6,67 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let formatData = sourceString.split(';').map((item) => item.split(':'));
+
+  formatData = formatData.map((item) => {
+    for (let i = 0; i < item.length; i++) {
+      if (item[i].includes('\n')) {
+        do {
+          item[i] = item[i].replace('\n', '');
+        } while (item[i].includes('\n'));
+      }
+    }
+
+    return item;
+  });
+
+  formatData = formatData.map((item) => item.map((el) => el.trim()));
+
+  formatData = formatData.filter((item) => item.length > 1);
```

Comment:
The filtering step correctly removes entries that do not have both a key and a value. This aligns with the task requirements.